template/lxr.json.ejs

```diff
@@ -2,6 +2,6 @@
   "host": "<%= host %>",
   "workspace": "<%= workspace %>",
   "apitoken": "<%= apitoken %>",
-  "proxyURL": "<%= proxyURL %>",
+  "proxyURL": "<%= behindProxy ? proxyURL : '' %>",
```

Comment:
Not sure if it would make sense to have a empty `proxyUrl`, but we could prevent that by:

```ejs
<% if (behindProxy) &>
  "proxyUrl": "<%= proxyUrl %>",
<% } %>
```

Reply:
I didn't give it too much thought, just tried to keep existing behavior. 😉 

Using an `if` to exclude empty properties would definitely generate a cleaner `lxr.json`. On the other hand, I found it handy in the past to have e.g. the `apitoken` key already present in the file so that I could simply add the value without having to look up whether it's `token` or `apiToken` or `apitoken`... What do you think?

Reply:
Fine for me! 👍